packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -156,8 +156,10 @@ const router = (
 export const handler = async (
   event: OriginRequestEvent | OriginResponseEvent
 ): Promise<CloudFrontResultResponse | CloudFrontRequest> => {
+  // @ts-ignore
```

Comment:
do we need these @ts-ignores ?

Reply:
Hm, I saw errors when building at one point due to this being from json file, maybe this one is not needed anymore, let me try to remove it